administrator/components/com_mails/src/Model/TemplatesModel.php

```diff
@@ -173,6 +173,25 @@ protected function getListQuery()
 		return $query;
 	}
 
+	/**
+	 * Get list of extensions which are using mail templates
+	 *
+	 * @return array
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getExtensions()
+	{
+		$db    = $this->getDbo();
+		$query = $db->getQuery(true)
+			->select('DISTINCT SUBSTRING(' . $db->quoteName('template_id')
+				. ', 1, POSITION(' . $db->quote('.') . ' IN ' . $db->quoteName('template_id') . ') - 1) AS ' .  $db->quoteName(extension))
```

Comment:
```suggestion
			->select(
				'DISTINCT SUBSTRING(' . $db->quoteName('template_id')
				. ', 1, POSITION(' . $db->quote('.') . ' IN ' . $db->quoteName('template_id') . ') - 1) AS ' .  $db->quoteName(extension)
			)
```
This should fix some of the PHP CS issues, if not all.